[Android] Build fail?

Hey @negativetwelve, thanks again for the package!

Working great on iOS; not integrating on Android but am getting a build error:

```
/Users/SLEE/Github/Aura/node_modules/react-native-ux-cam/android/src/main/java/com/rnuxcam/rnuxcam/UXCamModule.java:75: error: cannot find symbol
      Object value = properties.getObject(key);
                               ^
  symbol:   method getObject(String)
  location: variable properties of type ReadableMap
/Users/SLEE/Github/Aura/node_modules/react-native-ux-cam/android/src/main/java/com/rnuxcam/rnuxcam/UXCamModule.java:76: error: no suitable method found for put(String,Object)
      map.put(key, value);
         ^
    method Map.put(String,String) is not applicable
      (argument mismatch; Object cannot be converted to String)
    method AbstractMap.put(String,String) is not applicable
      (argument mismatch; Object cannot be converted to String)
    method HashMap.put(String,String) is not applicable
      (argument mismatch; Object cannot be converted to String)
2 errors
:react-native-ux-cam:compileReleaseJavaWithJavac FAILED
```

Am I missing something here?
"react-native-ux-cam": "^2.2.2",
"react-native": "^0.39.2",

Thanks!

--- a/android/src/main/java/com/rnuxcam/rnuxcam/UXCamModule.java
+++ b/android/src/main/java/com/rnuxcam/rnuxcam/UXCamModule.java
@@ -72,7 +72,7 @@ public class UXCamModule extends ReactContextBaseJavaModule {
     ReadableMapKeySetIterator iterator = properties.keySetIterator();
     while (iterator.hasNextKey()) {
       String key = iterator.nextKey();
-      Object value = properties.getObject(key);
+      String value = properties.getString(key);
       map.put(key, value);
     }
     UXCam.addTagWithProperties(tag, map);

